# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/cairo/hotfix.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- cairo-1.18.0/meson.build.vanilla	2023-09-23 19:21:54.707044050 +0200
+++ cairo-1.18.0/meson.build	2023-09-23 19:22:11.071045355 +0200
@@ -378,11 +378,7 @@
     ipc_rmid_deferred_release = (prop_str == 'true')
     message('IPC_RMID_DEFERRED_RELEASE:', ipc_rmid_deferred_release)
   elif prop_str == 'auto'
-    res = cc.run(files('meson-cc-tests/ipc_rmid_deferred_release.c'),
-      dependencies: [x11_dep, xext_dep],
-      name: 'shmctl IPC_RMID allowes subsequent attaches')
-
-    ipc_rmid_deferred_release = (res.returncode() == 0)
+    ipc_rmid_deferred_release = true
   else
     error('Unexpected value for external property ipc_rmid_deferred_release: @0@'.format(prop_str))
   endif
